### ISSUE TITLE
configure: error: could not find jni.h

### ISSUE THREAD

**Issue by AndCycle** [rdp/ffmpeg-windows-build-helpers#268]
there is a simple issue caused by environment var

```
Cloning into '/home/andcycle/ffmpeg-windows-build-helpers/sandbox/win64/libbluray_git/contrib/libudfread'...
Submodule path 'contrib/libudfread': checked out '131629921cc756c38eaf3e2d6b69ba2db690b199'
...
checking jni.h usability... no
checking jni.h presence... no
checking for jni.h... no
configure: error: could not find jni.h
```

due to `libbluray` use `JDK_HOME` var from environment, it failed to correct configure,
can be fixed by unset `JDK_HOME`,

I don't know if this kind of bug for upstream because there are `--disable-bdjava-jar` in `ffmpeg-windows-build-helpers`

**Comment by rdp** [comment]
hmm are you still running into this?  I could add an unset in there...

**Comment by rdp** [comment]
OK blind fix attempted:  a9873af

**Comment by rdp** [comment]
It might be nice for somebody to report this upstream as well...
